typescript-functions/Untitled-1.ts

```diff
@@ -0,0 +1,56 @@
+function convertMinutesToSeconds(x: number): number {
+  return x * 60;
+}
+
+const convertMinutesToSecondsResult: number = convertMinutesToSeconds(10); // 600
+console.log(
+  'value fo convertminutestosecondsresult:',
+  convertMinutesToSecondsResult
+);
+
+function greet(x: string): string {
+  return 'Hello' + ' ' + x;
+}
+
+const greetfunction: string = greet('Bryant');
+console.log('value of greetfunction : ', greetfunction);
+
+function getArea(x: number, y: number): number {
+  return x * y;
+}
+
+const getAreaResult: number = getArea(1, 2);
+console.log('value of getarearesult:', getAreaResult);
+
+interface people {
+  firstName: string;
+  lastName: string;
+}
+
+const person: people = {
+  firstName: 'Bryant',
+  lastName: 'Mendoza',
+};
+
+function getFirstName(person: object): string {
+  return person.firstName;
+}
```

Comment:
Couple of things here, first you should capitalize your `interface` name so that it is more clear `interface People`.

Second you typed your `person` with `People`, which is correct, but you typed your `person` parameter in your `getFirstName` function with type of `object`. This is not specific enough and we want this `person` to be of type `People`